### PR TITLE
Clean apm completions

### PR DESCRIPTION
**Pull request by tbhaxor**
## Description

Fixed the false-positive SC1091 warning in APM completion due to invalid `$BASH_IT` value

## How Has This Been Tested?

Locally on Linux

## Screenshots (if appropriate):

**Before**
image

**After**

Screenshot_20220313_040907

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] If my change requires a change to the documentation, I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] If I have added a new file, I also added it to ``clean_files.txt`` and formatted it using ``lint_clean_files.sh``.
- [ ] I have added tests to cover my changes, and all the new and existing tests pass.

### CODE REVIEW

File: completion/available/apm.completion.bash [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # shellcheck shell=bash
 about-completion "apm completion"
-# shellcheck disable=SC1090
+# shellcheck disable=SC1090,SC1091
```
[the remark this answers is not in the document]
Comment (tbhaxor) [reply completion/available/apm.completion.bash:3]: I didn't get this. Also, how would that solve the problem?

[the remark this answers is not in the document]
Comment (tbhaxor) [reply completion/available/apm.completion.bash:3]: @NoahGorny Ping

[the remark this answers is not in the document]
Comment (NoahGorny) [reply completion/available/apm.completion.bash:3]: If you use the shellcheck path directive, you wouldnt need to suppress any error

[the remark this answers is not in the document]
Comment (tbhaxor) [reply completion/available/apm.completion.bash]: @NoahGorny Fixed

**Comment by tbhaxor** [comment]
Timeout

**Pull request closed without merging**